configs/eslint-common-node.json

```diff
@@ -1,5 +1,19 @@
 {
-    "extends": ["plugin:node/recommended", "./eslint-common.json"],
+    "extends": [
+        "airbnb-base",
+        "./eslint-common.json",
+        "plugin:node/recommended"
+    ],
+    "rules": {
+        "node/no-unsupported-features/es-syntax": "off",
+        // NOTE: handled by import/no-extraneous-dependencies
+        "node/no-extraneous-require": "off"
+    },
```

Comment:
There are probably more rules which can be disabled as we are moving towards having fully TS codebase, but I think it's not for this PR.